### DISCUSSION TITLE
Repositório não exclui

### DISCUSSION THREAD

**Discussion by RodrigoAlvesF** [community/community#38627] [Other Feature Feedback, Questions, & Ideas] [answered]
Coloco meu link na hora de excluir e aperto o botão vermelho porém diz a mensagem no canto superior que o repositório foi excluido, mas não exclui, ele continua lá!
Alguém sabe como resolver?

**Comment by mdAshik922** [comment]
See if it works by reloading the browser

**Comment by ossanna16** [comment]
Hi there @RodrigoAlvesF and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by DSDmark** [comment, marked as answer]
Hi @RodrigoAlvesF,

Did really delete your **repo** like that inside that settings:-

![2022-11-10_16-39](https://user-images.githubusercontent.com/85744816/201076410-a9c7ec6c-f3f0-4372-9898-900d49021484.png)


then, can you write the link your repo

**Comment by RodrigoAlvesF** [comment]
Solved, i wait 1 day then done haha, maybe its a delay
Thanks friends!
